Articles/Setup/Install/install-umbraco-with-templates.md

```diff
@@ -0,0 +1,97 @@
+
+```none
+Templates                    Short Name               Language          Tags
+------------------------------------------------------------------------------------------------------
+Umbraco Solution             umbraco                  [C#]              Web/CMS/Umbraco
+Umbraco Package              umbracopackage           [C#]              Web/CMS/Umbraco/Package/Plugin
+```
+
+To get **help** on a project template with `dotnet new` run the following command:  
+`dotnet new umbraco -h`
+
+```none
+Usage: new [options]
+
+Options:
+  -h, --help          Displays help for this command.
+  -l, --list          Lists templates containing the specified name. If no name is specified, lists all templates.
+  -n, --name          The name for the output being created. If no name is specified, the name of the current directory is used.
+  -o, --output        Location to place the generated output.
+  -i, --install       Installs a source or a template pack.
+  -u, --uninstall     Uninstalls a source or a template pack.
+  --interactive       Allows the internal dotnet restore command to stop and wait for user input or action (for example to complete authentication).
+  --nuget-source      Specifies a NuGet source to use during install.
+  --type              Filters templates based on available types. Predefined values are "project", "item" or "other".
+  --dry-run           Displays a summary of what would happen if the given command line were run if it would result in a template creation.
+  --force             Forces content to be generated even if it would change existing files.
+  -lang, --language   Filters templates based on language and specifies the language of the template to create.
+  --update-check      Check the currently installed template packs for updates.
+  --update-apply      Check the currently installed template packs for update, and install the updates.
+
+
+Umbraco Solution (C#)
+Author: Umbraco HQ
+Description: An empty Umbraco Solution ready to get started
+Options:
+  -v|--version              The version of Umbraco to load using NuGet
+                            string - Optional
+                            Default: 9.0.0-beta001.20210407.7
+
+  -p|--PackageTestSiteName  The name of the package this should be a test site for (Default: '')
+                            text - Optional
+
+  -ce|--SqlCe               Adds the required dependencies to use SqlCE (Windows only) (Default: false)
+                            bool - Optional
+                            Default: false / (*) true
+
+  -F|--Framework            The target framework for the project.
+                                net5.0    - Target net5.0
+                                net6.0    - Target net6.0
+                            Default: net5.0
+
+
+* Indicates the value used if the switch is provided without a value.
+```
```

Comment:
What exactly is this snippet? Would it make sense to add an explanation or a short description above the snippet?

It might just be me, who's missing something, but as someone who doesn't know anything about how this all works, I was a bit confused about what I can use all the info in the snippet for 😅

Reply:
I totally understand :D 

Just above this selected snippet is the command that you need to execute to get the information from the snippet, but I will add some info that will hopefully make it clear 🙂

Reply:
Aaah! That makes sense 😄 Thanks for adding the additional notes 💪